styx-flyte-client/src/main/java/com/spotify/styx/flyte/client/FlyteAdminClient.java

```diff
@@ -0,0 +1,93 @@
+/*-
+ * -\-\-
+ * Spotify Styx Testing Utilities
```

Comment:
copy pasta?

Reply:
I thought that running the mvn license would fix that. 🤦